patches/sitecustomize.py

```diff
@@ -0,0 +1,23 @@
+# Monkey patches BigQuery client creation to use data proxy
```

Comment:
If we're going to monkey patch, can we at least have a test (see `test_build.py`) covering it?

Reply:
👍